0021-plugin-rpc-api/0021-plugin-rpc-api.md

```diff
@@ -0,0 +1,170 @@
+# Plugin RPC API
+
+## Description
+
+> TODO
+
+## Specification
+
+### Send Transfer
+
+#### Request
+
+```http
+POST /rpc/?method=send_transfer&prefix=peer.me HTTP/1.1
+Host: rpchost
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer ABCXYZ
+
+[
+  {
+    "id": "0e798bd6-213b-4b2f-bc1c-040788e7bae5",
+    "ledger": "peer.me.",
+    "to": "peer.me.Y_luxphkAy6ddYzuXb9lXxS60zg5tHjrPh8zz_BfwEA",
+    "amount": "348807",
+    "ilp": "ARwAAAAAB1TUwA5nLnVzLm5leHVzLmJvYgMEEEEA",
+    "executionCondition": "7td8LdXdYkv-6WXWdMlPZ1DhROwRFdazA0m3kTz4LUI",
+    "expiresAt": "2017-06-14T11:58:18.509Z",
+    "direction": "outgoing"
+  }
+]
+```
+
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+true
+```
+
+### Send Request
+
+#### Request
+
+```http
+POST /rpc/?method=send_request&prefix=peer.me HTTP/1.1
+Host: rpchost
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer ABCXYZ
+
+[
+  {
+    "ledger": "peer.me.",
+    "to": "peer.me.Y_luxphkAy6ddYzuXb9lXxS60zg5tHjrPh8zz_BfwEA",
+    "ilp": "ARwAAAAAB1TUwA5nLnVzLm5leHVzLmJvYgMEEEEA"
+  }
+]
+```
+
+#### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id": "0e798bd6-213b-4b2f-bc1c-040788e7bae5",
+  "
```

Comment:
Extra `"` here. Also, what does this `id` mean? Shouldn't there be something substantive in the response?